python-sdk/src/astro/dataframes/pandas.py

```diff
@@ -45,8 +45,8 @@ def deserialize(data: dict | str, version: int):
             if file.is_dataframe:
                 logger.info("Retrieving file from %s using %s conn_id ", file.path, file.conn_id)
                 return file.export_to_dataframe()
-            return File
-        return PandasDataframe.from_pandas_df(read_json(data))
+            return file
+        return PandasDataframe.from_pandas_df(PandasDataframe.from_dict(data))
```

Comment:
Should we add test for this change? coverage is complaining

Reply:
This is covered by https://github.com/astronomer/astro-sdk/blob/6e0b9eee84b96759ae0249575034aeb1a340edc5/python-sdk/tests/dataframes/pandas.py#L73